openhtf/util/logs.py

```diff
@@ -254,27 +290,31 @@ def format(self, record):
 
 
 @functions.call_once
-def configure_cli_logging():
-  """Configure OpenHTF to log to the CLI based on verbosity arg."""
+def initialize_logging():
```

Comment:
I'd like to keep the name `configure_logging` instead of `initialize_logging` for better parity with general Python logging system parlance. I added the "cli" in my logging refactor because I wanted it to be more immediately clear at call sites that this isn't where test record logging is set up.